huggingface/docker/Dockerfile_clm

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu115-py38-torch1110
+
+RUN python -m pip install \
+                        torch>=1.9.0 \
+                        datasets>=1.8.0 \
+                        sentencepiece>0.1.92 \
+                        transformers==4.16.0 \
+                        onnx>=1.9.0 \
+                        onnxruntime-training>=1.9.0 \
```

Comment:
are we sure using the onnxruntime-training from pypi is ok to run all the examples?

Reply:
I tried testing it a couple times and didn't encounter any issues -- would installing it from source or something else be a better approach?

Reply:
Another option is : https://download.onnxruntime.ai/

This has different variants for ort packages. Pypi only has: https://download.onnxruntime.ai/onnxruntime_stable_cu102.html

Reply:
Thanks Ashwini! I asked Frank for help on this & I realized the PTCA docker image that the Dockerfile is based on already installs the best ORT package suited for the CUDA version, so I removed the ORT and ONNX install from the Dockerfile_clm file.